File: WordGames/noEAplus15/noEAplus15.js
// Program to find all the words with no A's or E's and are at least 15 letters long
// Importing the fs module
let fs = require("fs")

// Intitializing the readFileLines with the file
const readFileLines = filename =>
   fs.readFileSync(filename)
   .toString('UTF8')
   .split('\n');

// Calling the readFiles function with file name
let words = readFileLines('/home/evergreen/UnderDogsAssignments/sowpods');
noEA = []
noEAPlus15 = []

for(let i = 0; i < words.length; i++) { 
    if (!words[i].includes("E") && !words[i].includes("A")) {
        noEA.push(words[i]);}}

for(let j = 0; j < noEA.length; j++) { 
    if (noEA[j].length >= 15 ) {
        noEAPlus15.push(noEA[j]);
    }}
console.log("The words containing no E's or A's and at least 15 letters long are " + noEAPlus15)

File: WordGames/noEAplus15/noEAplus15.py
# Program to create a list for all of the words that hhave no Es or As and are at least 15 letters long

# importing the word list
sow_pods = open("/home/evergreen/UnderDogsAssignments/sowpods")

# Go through each word in the file and add it to the list, words
words = []
noEA = []
noEA15 = []

for line in sow_pods:
    words.append(line.strip())

for w in words:
  if ("E" not in w) and ("A" not  in w):
      noEA.append(w)

for y in noEA:
    if len(y) >= 15:
        noEA15.append(y)
    
print('All of the words that have no As or Es and are at least 15 letters long are ' + str(noEA15))